app/controllers/admin/routes.py

```diff
@@ -143,6 +142,26 @@ def addRecurringEvents():
     recurringEvents = calculateRecurringEventFrequency(preprocessEventData(request.form.copy()))
     return json.dumps(recurringEvents, default=str)
 
+@admin_bp.route('/courseProposals', methods=['GET'])
+def createTable():
+    courseDict = getProposalData(g.current_user)
+    try:
+        return render_template("/admin/courseProposals.html",
+                                instructor = g.current_user,
+                                courseDict = courseDict)
+    except Exception as e:
+        print('Error while creating table: ', e)
+        return "", 500
+
+@admin_bp.route('/courseProposals/<courseID>/withdraw/', methods = ['POST'])
```

Comment:
Another case where a check of the user is needed.  Make sure a user can't mess around with urls and withdraw a course that isn't theirs.

Reply:
*Check to see if the user is a CELTS admin